common/trainer/default_trainer.py

```diff
@@ -153,29 +151,37 @@ def prepare(self, model, tokenizer, dataset, optimizer, accelerator):
     def train(self):
         num_train_epochs = self.config.get("num_train_epochs", 1)
         checkpoint = self.config.get("checkpoint")
-        log_step = self.config.get("log_step", 1)
+        logging_steps = self.config.get("logging_steps", 1)
         max_train_step = self.config.get("max_train_step")
         max_eval_step = self.config.get("max_eval_step")
+        gradient_accumulation_steps = self.accelerator.gradient_accumulation_steps
+        output = self.config.get("output", "./output")
+        writer = torch.utils.tensorboard.SummaryWriter(output)
```

Comment:
I don’t think the `writer` is needed, maybe ray.train.report is enough. After running `tensorboard --logdir .../ray_results/TorchTrainer_2024* --bind_all` we can see the parameters set in `report()`. @carsonwang please help confirm this.